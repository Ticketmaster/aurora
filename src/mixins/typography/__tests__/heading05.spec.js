import HEADING_05 from "../heading05";

describe("HEADING_05", () => {
  it("should return the correct values", () => {
    expect(HEADING_05).toMatchSnapshot();
  });
});
