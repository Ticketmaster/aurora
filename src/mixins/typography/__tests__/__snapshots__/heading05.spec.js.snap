// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HEADING_05 should return the correct values 1`] = `
Array [
  "
  ",
  "
  color: ",
  [Function],
  ";
",
  ";
  ",
  "
  font-size: ",
  "14px",
  ";
",
  ";
  ",
  "
  ",
  "font-weight: 600",
  ";
",
  ";
  ",
  "
  line-height: ",
  "1.25",
  ";
",
  ";
",
]
`;
